docker/rumble/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Kerberos
+keytab=/etc/rumble/keytab
+principal=`klist -k "$keytab" | tail -1 | awk '{print $2}'`
+echo "principal=$principal"
+kinit $principal -k -t "$keytab"
+if [ $? == 1 ]; then
+    echo "Unable to perform kinit"
+    exit 1
+fi
+klist -k "$keytab"
+
+git clone https://github.com/mrceyhun/CMSMonitoring.git && \
```

Comment:
why do you need to build rumble server in run.sh script?

Reply:
In order to get latest CMSMonitoring changes in each k8s deployment. When we build rumble_server.go  in Dockerfile, we need to rebuild docker image to get latest changes in CMSMonitoring.

Reply:
No, it is wrong. If you need new code it means you should build new image. Moreover each build should come from a branch and not head of the master. Doing this way you can reproduce the behaviour of the image. Please remove code build from run script. Add a tag into Dockerfile and use it. See example in [here](https://github.com/dmwm/CMSKubernetes/blob/master/docker/das-server/Dockerfile). Then in CI/CD the tag can be substituted to a new one and everything can be automated, see [das CI/CD](https://github.com/dmwm/das2go/blob/master/.github/workflows/build.yml#L73)

Reply:
@vkuznet 
I removed go build from run script, tested and deployed my test cluster. If it's fine, I would like to postpone CI/CD implementation for a few days. I can open another PR for CI/CD.